src/Symfony/Component/Security/Http/Event/InteractiveLoginFailureEvent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+/**
+ * InteractiveLoginFailureEvent.
```

Comment:
The other classes in this part of Symfony are probably not a good template for our code style. They just haven't been touch for a long time so that's why they may look a bit different.